README.md

```diff
@@ -367,12 +377,14 @@ These DataUnion-specific options are used from `StreamrClient` options:
 Here's an example how to deploy a data union contract and set the admin fee:
 
 ```js
+import { StreamrClient } from 'streamr-client'
+
 const client = new StreamrClient({
     auth: { privateKey },
 })
 
 const dataUnion = await client.deployDataUnion()
-await client.setAdminFee(0.3, { dataUnion })
+const receipt = await dataUnion.setAdminFee(0.3, { dataUnion })
```

Comment:
The second parameter `(({ dataUnion })` has been removed

Reply:
it has in fact been removed  ;)  https://github.com/streamr-dev/streamr-client-javascript/pull/224

Reply:
or about to be

Reply:
In #224 you remove option for another method (`addMembers`, `removeMembers`), not `setAdminFee`.

Reply:
Fixed here